draft-ietf-quic-manageability.md

```diff
@@ -863,11 +907,167 @@ security-relevant.
 
 Dan Druta contributed text to {{sec-ddos-dec}}. Igor Lubashev contributed text
 to {{sec-loadbalancing}} on the use of the connection ID for load balancing.
-Marcus Ilhar contributed text to {{sec-rtt}} on the use of the spin bit.
+Marcus Ilhar contributed text to {{sec-rtt}} on the use of the spin bit. The 
+pseudo provided in the appendix is based on input provided by David Schinazi.
 
 # Acknowledgments
 
+Thanks to Martin Thomas and Martin Duke for contributing by reviewing and 
+providing text proposals.
+
 This work is partially supported by the European Commission under Horizon 2020
 grant agreement no. 688421 Measurement and Architecture for a Middleboxed
 Internet (MAMI), and by the Swiss State Secretariat for Education, Research, and
 Innovation under contract no. 15.0268. This support does not imply endorsement.
+
+# Appendix
+
+## Handling of Google QUIC  {#sec-google-version} 
+
+This sections contains algorithms that allows parsing versions from both
+Google QUIC and IETF QUIC. These mechanisms will become
+irrelevant when IETF QUIC is fully deployed and Google QUIC is deprecated.
```

Comment:
Add something here noting that the rest of this document says nothing about Google QUIC.